LoopBackTests/LBUserTests.m

```diff
@@ -81,7 +81,7 @@ - (void)setUp {
     NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
     [defaults removeObjectForKey:DEFAULTS_CURRENT_USER_ID_KEY];
 
-    self.adapter = [LBRESTAdapter adapterWithURL:[NSURL URLWithString:@"http://localhost:3000"]];
+    self.adapter = [LBRESTAdapter adapterWithURL:[NSURL URLWithString:@"http://localhost:3000/api"]];
```

Comment:
My bad, I forgot to change it back! Is there a LoopBack app out there you guys use for testing the SDK?